frontend/packages/knative-plugin/src/utils/knative-topology-utils.ts

```diff
@@ -275,51 +252,51 @@ export const tranformKnNodeData = (
   operatorBackedServiceKinds: string[],
   utils?: Function[],
   cheURL?: string,
-  application?: string,
 ) => {
   let nodesData = [];
   let edgesData = [];
+  let groupsData = [];
   const dataToShowOnNodes = {};
-  const resourceData = filterKnativeBasedOnActiveApplication(knResourcesData, application);
-  _.forEach(resourceData, (res) => {
+  _.forEach(knResourcesData, (res) => {
     const uid = _.get(res, ['metadata', 'uid']);
```

Comment:
Any chance uid or metadata is not present? IMO, metadata and uid are going to be defined always and we can directly use dot notation here. If not, Please add a third param(default value) to `_.get`

Reply:
it'll always be there, dot will make sense